CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.21)
 
 
 set(ZITI_SDK_DIR "" CACHE FILEPATH "developer option: use local ziti-sdk-c checkout")
-set(ZITI_SDK_VERSION "1.5.4" CACHE STRING "ziti-sdk-c version or branch to use")
```

Comment:
this intentional?

Reply:
yes, until the needed csdk changes are tagged/released